tests/testthat/test-tab_options.R

```diff
@@ -1463,17 +1463,19 @@ test_that("the row striping options work correctly", {
       selection_text("[class='gt_row gt_left gt_stub gt_striped']"),
     0)
 
+  # TODO: determine why this doesn't work as expected
+
```

Comment:
?

Reply:
This is a real, but unrelated bug. Using `tbl %>% tab_options(row.striping.include_stub = TRUE)` no ops. Will file an issue for this.